src/main/java/domain/Division.java

```diff
@@ -0,0 +1,8 @@
+package domain;
+
+public class Division implements Operator {
+    @Override
+    public Number operate(int first, int second) {
+        return new Number(first / second);
+    }
```

Comment:
```suggestion
    public Number operate(Number first, Number second) {
        return first.division(second);
    }
```
이와 같이 int가 아닌 Number 객체 단위로 접근해 보면 어떨까?
질문 3에 대한 힌트가 되지 않을까?